webpack.config.babel.js

```diff
@@ -37,12 +37,17 @@ const productionPlugins = [
     // this is actually the property name https://github.com/kimhou/replace-bundle-webpack-plugin/issues/1
     partten: /throw\s+(new\s+)?[a-zA-Z]+Error\s*\(/g,
     replacement: () => 'return;('
-  }])
+  }]),
+
+  new CopyWebpackPlugin([
+    { from: './dropdown-arrow-down.png', to: 'dropdown-arrow-down.png' }
```

Comment:
SVG is probably the answer - just a shame I can't use it as background image and have to pollute the markup with something which isn't that useful for screenreaders